src/main/java/sorald/Main.java

```diff
@@ -4,6 +4,12 @@
 
 public class Main {
     public static void main(String[] args) {
+        try {
+            throw new InterruptedException();
+        } catch (InterruptedException | IllegalArgumentException e) {
+            System.out.println("oops!");
```

Comment:
To fix violation '2142:sorald/Main.java:9:17:9:66', Sorald suggests the following:
```suggestion
$(hunk.additions.join('
'))
```